### PR TITLE
Add memcached-for-Drupal 8 instructions

### PR DESCRIPTION
**Pull request by Crell**
Developed from working with a customer who was trying to narrow down some odd cache-related issues.  Not recommended but no sense not documenting it.

### CODE REVIEW

File: src/frameworks/drupal8/memcached.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  }
+}
+```
+
+The `example.services.yml` file noted above will also use Redis for the lock and flood
+control systems.
+
+The redis module is able to use Redis as a queue backend, however, that should not be done on an ephemeral Redis instance as that could result in lost items when the Redis service instance is restarted or fills up.  If you wish to use Redis for the queue we recommend using a separate persistent Redis instance.  See the [Redis documentation page](/configuration/services/redis.md) for more information.
+
+### Verifying Redis is running
```
Comment (ralt) [inline src/frameworks/drupal8/memcached.md]: ???

File: src/frameworks/drupal8/memcached.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+```
+
+That will create a service named `cache`, of type `memcached`, specifically version `1.4`.
+
+### Expose the Memcached service to your application
+
+In your `.platform.app.yaml` file, we now need to open a connection to the new Redis service.  Under the `relationships` section, add the following:
+
+```yaml
+relationships:
```
Comment (ralt) [inline src/frameworks/drupal8/memcached.md]: Choose a different name for the key?

Comment (Crell) [reply src/frameworks/drupal8/memcached.md]: New name chosen.

File: src/frameworks/drupal8/memcached.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    cache: "cacheservice:memcached"
+```
+
+The key (left side) is the name that will be exposed to the application in the `PLATFORM_RELATIONSHIPS` [variable](/development/variables.md).  The right hand side is the name of the service we specified above (`cache`) and the endpoint (`memcached`).  If you named the service something different above, change `cache` to that.
```
Comment (ralt) [inline src/frameworks/drupal8/memcached.md]: > change `cacheservice` to that.

**Pull request merged**